templates/ocs-deployment/operator-openshift-with-csi.yaml

```diff
@@ -0,0 +1,162 @@
+  - driver: "ceph.rook.io/rook"
+  - driver: "ceph.rook.io/rook-ceph"
+volumes:
+  - configMap
+  - downwardAPI
+  - emptyDir
+  - flexVolume
+  - hostPath
+  - persistentVolumeClaim
+  - projected
+  - secret
+users:
+  # A user needs to be added for each rook service account.
+  # This assumes running in the default sample "rook-ceph" namespace.
+  # If other namespaces or service accounts are configured, they need to be updated here.
+  - system:serviceaccount:{{ cluster_namespace | default('openshift-storage') }}:rook-ceph-system
+  - system:serviceaccount:{{ cluster_namespace | default('openshift-storage') }}:default
+  - system:serviceaccount:{{ cluster_namespace | default('openshift-storage') }}:rook-ceph-mgr
+  - system:serviceaccount:{{ cluster_namespace | default('openshift-storage') }}:rook-ceph-osd
+---
+kind: SecurityContextConstraints
+# older versions of openshift have "apiVersion: v1"
+apiVersion: security.openshift.io/v1
+metadata:
+  name: rook-ceph-csi
+allowPrivilegedContainer: true
+allowHostNetwork: true
+allowHostDirVolumePlugin: true
+priority:
+allowedCapabilities: ['*']
+allowHostPorts: true
+allowHostPID: true
+allowHostIPC: true
+readOnlyRootFilesystem: false
+requiredDropCapabilities: []
+defaultAddCapabilities: []
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+fsGroup:
+  type: RunAsAny
+supplementalGroups:
+  type: RunAsAny
+allowedFlexVolumes:
+  - driver: "ceph.rook.io/rook"
+  - driver: "ceph.rook.io/rook-ceph"
+volumes: ['*']
+users:
+  # A user needs to be added for each rook service account.
+  # This assumes running in the default sample "rook-ceph" namespace.
+  # If other namespaces or service accounts are configured, they need to be updated here.
+  - system:serviceaccount:{{ cluster_namespace | default('openshift-storage') }}:rook-csi-rbd-plugin-sa
+  - system:serviceaccount:{{ cluster_namespace | default('openshift-storage') }}:rook-csi-rbd-provisioner-sa
+  - system:serviceaccount:{{ cluster_namespace | default('openshift-storage') }}:rook-csi-rbd-attacher-sa
+  - system:serviceaccount:{{ cluster_namespace | default('openshift-storage') }}:rook-csi-cephfs-plugin-sa
+  - system:serviceaccount:{{ cluster_namespace | default('openshift-storage') }}:rook-csi-cephfs-provisioner-sa
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rook-ceph-operator
+  namespace: {{ cluster_namespace | default('openshift-storage') }}
+  labels:
+    operator: rook
+    storage-backend: ceph
+spec:
+  selector:
+    matchLabels:
+      app: rook-ceph-operator
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: rook-ceph-operator
+    spec:
+      serviceAccountName: rook-ceph-system
+      containers:
+      - name: rook-ceph-operator
+        image: rook/ceph:master
+        args: ["ceph", "operator"]
+        volumeMounts:
+        - mountPath: /var/lib/rook
+          name: rook-config
+        - mountPath: /etc/ceph
+          name: default-config-dir
+        env:
```

Comment:
I am looking here that I parametrized more options here:
https://github.com/red-hat-storage/ocs-ci/blob/master/templates/ocs-deployment/operator-openshift.yaml#L124-L156

Can you please go over those and think which one is worth to parametrize here?
I see that this one is missing:
ROOK_LOG_LEVEL , and can be useful, can we add?

Reply:
@vavuthu did you check if possible to pass ROOK_LOG_LEVEL here? I think that should work.

Reply:
@petr-balogh : I have check the rook master and it didn't have these extra parameters which was there in old flex yamls. I am not sure those are removed as part of CSI or missed it out . I am going to raise upstream issue regarding the difference. 
As of now, I am going with default yaml file.

Reply:
Raised Issue in Rook to get clarification: https://github.com/rook/rook/issues/3211